### ISSUE TITLE
im.OnDuplicateKeyUpdate does not separate the columns to update with a comma - MySQL

### ISSUE THREAD

**Issue by 806software** [stephenafamo/bob#14]
**Version: v0.16.0**

As the title, for the mysql `im.OnDuplicateKeyUpdate`, the generated query does not separate the columns to update with a comma. The query is therefore invalid and gives an error:

 `You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near col_3 = ?` 

as there is no separating column after the `col_2` assignment.

Example (assuming col_1 is a key field):
```
query := mysql.Insert(
    im.Into(mysql.Quote("my_table"), "col_1", "col_2, "col_3"),
    im.Values(mysql.Arg("val 1", "val 2", "val 3")),
    im.OnDuplicateKeyUpdate.SetCol("col_2", mysql.Arg("val 2")).SetCol("col_3", mysql.Arg("val_3")),
)

q, _, _ := query.Build()

fmt.Println(q)
```

The above results in the following query:

```
INSERT INTO `my_table` (`col_1`, `col_2`, `col_3`) 
VALUES (?, ?, ?)
ON DUPLICATE KEY UPDATE
`col_2` = ?
`col_3` = ?
```

The expected query would be:

```
INSERT INTO `my_table` (`col_1`, `col_2`, `col_3`) 
VALUES (?, ?, ?)
ON DUPLICATE KEY UPDATE
`col_2` = ?,
`col_3` = ?
```

The assumption is also that I am using this mod correctly.

**Comment by stephenafamo** [comment]
Thanks for reporting this. It is now fixed in `v0.17.0`

**Comment by 806software** [comment]
Really appreciate the fast response!
